client/src/controllers/DismissibleController.ts

```diff
@@ -0,0 +1,56 @@
+import { Controller } from '@hotwired/stimulus';
+import { WAGTAIL_CONFIG } from '../config/wagtailConfig';
+
+/**
+ * <section
```

Comment:
```suggestion
 * Adds the ability to make an element dismissible so that it updates it's class and makes an async request.
 *
 * @example
 * <section
```

let's flesh this out a bit.